### PR TITLE
mrc-2243 Create runReportTabs Vue component

### PR DESCRIPTION
**Pull request by Nicolas-Dolan**
Moves `runReport` (and eventually report logs) from an ftl template to a parent Vue component `runReportTabs`.

### CODE REVIEW

File: src/app/static/src/js/components/runReport/runReportTabs.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                <div class="sidebar pb-4 pb-md-0">
+                    <nav class="pl-0 pr-0 pr-md-4 navbar navbar-light">
+                        <button type="button" class="d-md-none navbar-toggler" data-toggle="collapse"
+                                data-target="#sidebar">
+                            <span class="navbar-toggler-icon"></span>
+                        </button>
+                        <div class="d-md-block mt-4 mt-md-0 collapse navbar-collapse" id="sidebar">
+                            <ul class="nav flex-column list-unstyled mb-0">
+                                <li class="nav-item">
+                                    <a id="run-link" class="nav-link active" data-toggle="tab" role="tab" @click="switchTab('runReport')">Run a report</a>
```
Comment (mwoodbri) [inline src/app/static/src/js/components/runReport/runReportTabs.vue:14]: Best to add an `href="#"`here - the usual link hover pointer doesn't display on Firefox without an `href`

File: src/app/static/src/js/components/runReport/runReportTabs.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                                </li>
+                                <li class="nav-item">
+                                    <a id="logs-link" class="nav-link" data-toggle="tab" role="tab" @click="switchTab('reportLogs')">Report logs</a>
```
Comment (mwoodbri) [inline src/app/static/src/js/components/runReport/runReportTabs.vue:17]: And here

**Review by mwoodbri — CHANGES_REQUESTED** [review]
Looking very good!

One trivial change, and you'll also need to look at Selenium tests: in particular [this one](https://github.com/vimc/orderly-web/blob/mrc-2243/src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunReportPageTests.kt#L84) which is cause of build failure.

**Review by EmmaLRussell — COMMENTED** [review]
Code looks good to me, as Mark says, just the cursor issue and tests to sort out.

File: src/app/static/src/js/components/runReport/runReportTabs.vue (hunk 1 above)
Comment (EmmaLRussell) [reply src/app/static/src/js/components/runReport/runReportTabs.vue]: Chrome too..

**Review by mwoodbri — CHANGES_REQUESTED** [review]
Please just remove `chromedriver_linux64.zip` and this will be ready to merge

**Pull request merged**